airflow/migrations/versions/0050_3_0_0_add_dagbundlemodel.py

```diff
@@ -48,12 +49,32 @@ def upgrade():
         sa.Column("last_refreshed", UtcDateTime(timezone=True), nullable=True),
         sa.PrimaryKeyConstraint("name", name=op.f("dag_bundle_pkey")),
     )
+
     with op.batch_alter_table("dag", schema=None) as batch_op:
         batch_op.add_column(sa.Column("bundle_name", sa.String(length=250), nullable=True))
         batch_op.add_column(sa.Column("bundle_version", sa.String(length=200), nullable=True))
+
         batch_op.create_foreign_key(
             batch_op.f("dag_bundle_name_fkey"), "dag_bundle", ["bundle_name"], ["name"]
         )
+
+        conn = op.get_bind()
+        conn.execute(
+            text(
+                """
+                UPDATE dag
+                SET bundle_name =
+                    CASE
+                        WHEN fileloc LIKE '%/airflow/airflow/example_dags/%' THEN 'example_dags'
+                        ELSE 'dags-folder'
```

Comment:
@jedcunningham what do you think of this? I guess any string would work here instead of having the case statement. Dag processor will later update the dag on the next run.